dev/com.ibm.ws.concurrent.persistent_fat_demo_timers/test-application/demotimers/src/ejb/timers/AutomaticDatabase.java

```diff
@@ -88,30 +88,54 @@ private void initTable() throws SQLException {
             DatabaseMetaData md = conn.getMetaData();
             ResultSet rs = md.getTables(null, null, "AUTOMATICDATABASE", null);
             while (rs.next()) {
-                if (rs.getString("TABLE_NAME").equalsIgnoreCase("AUTOMATICDATABASE"))
+                if (rs.getString("TABLE_NAME").equalsIgnoreCase("AUTOMATICDATABASE")) {
+                    System.out.println("Found table AUTOMATICDATABASE. Skipping creation.");
                     return;
+                }
             }
 
             //If not, create it.
             try (Statement stmt = conn.createStatement()) {
                 stmt.execute(createTable);
-            }
+            } //Let initTableAndRow catch error
+
+            System.out.println("Created table AUTOMATICDATABASE");
         }
     }
 
     private void initRow() throws SQLException {
+        final String checkRow = "SELECT COUNT(*) FROM AUTOMATICDATABASE WHERE name = ?";
         final String createRow = "INSERT INTO AUTOMATICDATABASE VALUES(?,?)";
 
         //create count
         count = 0;
 
         try (Connection conn = ds.getConnection()) {
+            //See if row already exists
+            try (PreparedStatement pstmt = conn.prepareStatement(checkRow)) {
+                pstmt.setString(1, name);
+                ResultSet rs = pstmt.executeQuery();
+                //If the count of rows is more than 0 then set this true.
+                //Otherwise, set to false when row is not found, or 0 is returned as a result.
+                boolean exists = rs.next() ? rs.getInt(1) > 0 : false;
+                if (exists) {
+                    System.out.println("Found row identified by " + name + " in table AUTOMATICDATABASE.  Skipping creation.");
+                    return;
+                }
+            } catch (SQLException sqle) {
+                System.out.println("Caught exception attempting to find row identified by " + name + " in table AUTOMATICDATABASE."
+                                   + " Continuing on to attempt to create row.");
```

Comment:
I don't see why we should suppress this error.  If something goes wrong, we should let the test fail and report the problem.